Extra semicolons causing compilation erros

The cpp/generated/hdf5/protocols.cc does not compile under gcc 9.4 with `-Werror=pedantic`, due to extra semicolons. 

There are many in the file, but the first one appears at line 242.

The error message reads:

`/home/dch/src/mrd2/cpp/generated/hdf5/protocols.cc:242:2: error: extra ‘;’ [-Werror=pedantic]
  242 | };
         |  ^`

--- a/.devcontainer/Dockerfile
+++ b/.devcontainer/Dockerfile
@@ -89,7 +89,7 @@ RUN mkdir -p /home/vscode/.local/share/CMakeTools \
     && chown vscode:conda /home/vscode/.local/share/CMakeTools/cmake-tools-kits.json
 
 # Install the yardl tool
-ARG YARDL_VERSION=0.2.2
+ARG YARDL_VERSION=0.2.3
 RUN wget --quiet "https://github.com/microsoft/yardl/releases/download/v${YARDL_VERSION}/yardl_${YARDL_VERSION}_linux_x86_64.tar.gz" \
     && tar -xzf "yardl_${YARDL_VERSION}_linux_x86_64.tar.gz" \
     && mv yardl "/opt/conda/envs/${CONDA_ENVIRONMENT_NAME}/bin/" \

